go/vt/vttablet/tabletserver/query_executor.go

```diff
@@ -444,7 +444,7 @@ func (qre *QueryExecutor) checkAccess(authorized *tableacl.ACLResult, tableName
 		}
 
 		if qre.tsv.qe.strictTableACL {
-			errStr := fmt.Sprintf("table acl error: %q %v cannot run %v on table %q", callerID.Username, callerID.Groups, qre.plan.PlanID, tableName)
+			errStr := fmt.Sprintf("%s command denied to user '%s' for table '%s' (ACL check error)", qre.plan.PlanID.String(), callerID.Username, tableName)
```

Comment:
This is dropping the `groups` information for a given user when the ACL check fails. 
We should consider adding a user's groups if its not an empty set. 

```suggestion
			errStr := fmt.Sprintf("%s command denied to user '%s' for table '%s' (ACL check error)", qre.plan.PlanID.String(), callerID.Username, tableName)
			if len(callerID.Groups)) > 0 {
						errStr := fmt.Sprintf("%s command denied to user '%s' , in groups [%s], for table '%s' (ACL check error)", qre.plan.PlanID.String(), callerID.Username,  strings.Join(callerID.Groups, ","), tableName)

			}
```

Reply:
Applied the change via aa90f25